nautobot/extras/management/commands/validate_models.py

```diff
@@ -0,0 +1,70 @@
+from django.apps import apps
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+from django.core.exceptions import ValidationError
+
```

Comment:
Might make more sense to put this in `nautobot.core.management` instead of `nautobot.extras.management` as it doesn't really rely on anything in the `extras` app.